wdl-ast/src/v1/workflow.rs

```diff
@@ -117,10 +119,10 @@ impl<N: TreeNode> WorkflowDefinition<N> {
         self.metadata()
             .and_then(|s| {
                 s.items().find_map(|i| {
-                    if i.name().text() == "allowNestedInputs" {
+                    if i.name().text() == WORKFLOW_HINT_ALLOW_NESTED_INPUTS_ALIAS {
```

Comment:
@peterhuene I just skimmed the spec and couldn't find an answer. In WDL v1.2, are "hints" allowed to be in the meta section at all? Currently, if no form of the allow nested inputs hint is found _in the `hints` section_, it falls through and searches _only_ for the `allowNestedInputs` form. Either both forms should be usable in meta sections or neither. The current implementation is odd.

Reply:
It's entirely a backwards compatibility fallback and as `allow_nested_inputs` was never respected in the meta in 1.1, we only look for `allowNestedInputs` there.

Plus, `allowNestedInputs` in the meta section is now deprecated by 1.2 and its use as a fallback will be removed in the future.

Where possible, we should be encouraging `allow_nested_inputs` in a `hints` section from 1.2+.